README.md

```diff
@@ -1,2 +1,30 @@
 # hw01
 Repository for homework 01
+
+Hong Jin Jo
+====================
+
+Auto-biography
+---------------------
+
+This is a brief auto-biography of Hong Jin Jo.
+
+<img src="https://github.com/hongjinjo/myrepo/blob/master/JO_HONGJIN.jpg?raw=true">
+
+**Hong Jin Jo** was born in Seoul, Korea in 1988. He has attended the following schools:
+
```

Comment:
It's good  that you keep in mind to leave a blank line after each sentence. This makes the whole thing tidy and easy to read.